VariationalBayes/ExponentialFamilies.py

```diff
@@ -117,6 +117,48 @@ def get_e_log_dirichlet(alpha):
     return sp.special.digamma(alpha) - digamma_sum
 
 
+def get_e_fun_logitnormal(lognorm_means, lognorm_infos, \
```

Comment:
Please put in the function's descriptive comments how to get the ```gh_weights``` and ```gh_loc``` so the user is sure to get the right kind of Gauss-Hermite weights.